src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -38,6 +38,27 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code substring}.
+     *   Ignores case, and the substring need not be a full word.
+     *   <br>examples:<pre>
+     *       containsIgnoreCase("ABc def", "abc") == true
+     *       containsIgnoreCase("ABc def", "DEF") == true
+     *       containsIgnoreCase("ABc def", "AB") == true
+     *       </pre>
+     * @param sentence cannot be null
+     * @param substring cannot be null, cannot be empty
+     */
+    public static boolean containsIgnoreCase(String sentence, String substring) {
+        requireNonNull(sentence);
```

Comment:
Can also use `static seedu.address.commons.util.CollectionUtil.requireAllNonNull` for multiple requireNonNull's